### ISSUE TITLE
Calling actor method passed as argument from other actor segfaults

### ISSUE THREAD

**Issue by plajjan** [actonlang/acton#109]
### Code / what did you do

I tried this code:

```Acton
actor Foo():
    def func(a):
        print("request", a)

actor Bar(f):
    f("test")

actor main(env):
    var f = Foo()
    var b = Bar(f.func)
```

### Expected outcome

I expected it to work, seeing "request test" in my terminal.
```
request test
```

### Actual outcome

I got a segfault instead

```
Segmentation fault
```

I can instead pass the actor and access the method more explicitly, which then works:
```
actor Foo():
    def func(a):
        print("request", a)

actor Bar(f):
    f.func("test")

actor main(env):
    var f = Foo()
    var b = Bar(f)
```

### Meta
This is on the main branch
`actonc --version`:
```
acton 0.4.2.20210820.11.49.39
```

**Comment by nordlander** [comment]
The bug is isolated to the Deactorizer module. I'll give this highest prio.

**Comment by nordlander** [comment]
Resolved by #129.
